### PR TITLE
i18n(zh-cn): translate guides/integrations-guide/preact.mdx

### PR DESCRIPTION
**Pull request by lu-sd**
#### What kind of changes does this PR include?
- Translated content

#### Description

 - Translate guides/integrations-guide/preact.mdx to Chinese.

### CODE REVIEW

**Comment by lu-sd** [comment]
@liruifengv

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+##  贡献 
+
+这个包由 Astro 的核心团队维护。欢迎你提交问题或（PR）！
+
+## 更新日志
+
+查看 [CHANGELOG.md](https://github.com/withastro/astro/tree/main/packages/integrations/preact/CHANGELOG.md) 了解此集成的更改历史。
+
+[astro-integration]: /en/guides/integrations-guide/
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:142]:
```suggestion
[astro-integration]: /zh-cn/guides/integrations-guide/
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+[astro-ui-frameworks]: /en/core-concepts/framework-components/#using-framework-components
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:144]:
```suggestion
[astro-ui-frameworks]: /zh-cn/core-concepts/framework-components/#使用框架组件
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+##  示例
+
+*   在 [Astro Preact example](https://github.com/withastro/astro/tree/latest/examples/framework-preact) 中展示了如何在 Astro 项目中使用交互式的 Preact 组件。
+*   在 [Astro Nanostores example](https://github.com/withastro/astro/tree/latest/examples/with-nanostores) 中展示了如何在 Astro 项目中在不同组件之间共享状态，甚至可以跨不同框架共享状态。
+
+## 问题排查
+
+如果需要帮助，请访问 [Discord](https://astro.build/chat) 上的 `#support` 频道。我们友好的支持团队成员将在那里帮助你！
+
+你也可以查阅Astro集成文档 [Astro Integration Documentation][astro-integration] 获得更多集成信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:132]:
```suggestion
你也可以查阅 [Astro 集成文档][astro-integration] 获得更多集成信息。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+```sh
+# 使用 NPM
+npx astro add preact
+# 使用 Yarn
+yarn astro add preact
+# 使用 PNPM
+pnpm astro add preact
+```
+
+如果遇到任何问题，[请随时在GitHub留言](https://github.com/withastro/astro/issues) 并尝试以下的手动安装步骤。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:40]:
```suggestion
如果遇到任何问题，[请随时在 GitHub 留言](https://github.com/withastro/astro/issues) 并尝试以下的手动安装步骤。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+export default defineConfig({
+  // ...
+  integrations: [preact()],
+});
+```
+
+##  使用
+
+要在 Astro 中使用你的第一个 Preact 组件，请前往我们的 [UI framework documentation][astro-ui-frameworks]. 你将会发现：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:73]:
```suggestion
要在 Astro 中使用你的第一个 Preact 组件，请前往我们的 [UI 框架文档][astro-ui-frameworks]。你将会了解到：
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+*   📦 框架组件是如何加载的，
+*   💧 客户端水合选项以及，
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:76]:
```suggestion
*   💧 客户端水合选项
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  // ...
+  integrations: [preact()],
+});
+```
+
+##  使用
+
+要在 Astro 中使用你的第一个 Preact 组件，请前往我们的 [UI framework documentation][astro-ui-frameworks]. 你将会发现：
+
+*   📦 框架组件是如何加载的，
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:75]:
```suggestion
*   📦 如何加载框架组件
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+*   💧 客户端水合选项以及，
+*   🤝 将不同框架混合和嵌套在一起的机会
+
+还可以查阅我们的 [Astro Integration Documentation][astro-integration] 以获取更多关于集成的信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:79]:
```suggestion
还可以查阅我们的 [Astro 集成文档][astro-integration] 以获取更多关于集成的信息。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import preact from '@astrojs/preact';
+
+export default defineConfig({
+  integrations: [preact({ compat: true })],
+});
+```
+
+当启用了 `compat` 选项后，Preact 集成将会在你的项目中渲染 React 组件和 Preact 组件，同时还允许你在 Preact 组件中导入 React 组件。在 Preact 网站的“从 React 切换到 Preact”页面中可以阅读更多信息 [“Switching to Preact (from React)”](https://preactjs.com/guide/v10/switching-to-preact) 。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:103]:
```suggestion
当启用了 `compat` 选项后，Preact 集成将会在你的项目中渲染 React 组件和 Preact 组件，同时还允许你在 Preact 组件中导入 React 组件。在 Preact 网站的[“从 React 切换到 Preact”](https://preactjs.com/guide/v10/switching-to-preact)页面中可以阅读更多信息。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+    "react": "npm:@preact/compat@latest",
+    "react-dom": "npm:@preact/compat@latest"
+  }
+}
+```
+
+可以查阅[`pnpm` overrides](https://pnpm.io/package_json#pnpmoverrides) 和 [`yarn` resolutions](https://yarnpkg.com/configuration/manifest#resolutions) 的文档，了解它们各自的覆盖功能。
+
+:::note 
+目前，`compat` 选项只适用于 React 库,将代码以 ESM 的形式导出。如果在构建时出现错误，请尝试将该库添加到你的`vite.ssr.noExternal: ['the-react-library']` 的 `astro.config.mjs` 文件中.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:120]:
```suggestion
目前，`compat` 选项只适用于 React 库，将代码以 ESM 的形式导出。如果在构建时出现错误，请尝试将该库添加到你的`vite.ssr.noExternal: ['the-react-library']` 的 `astro.config.mjs` 文件中。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+:::
+
+##  示例
+
+*   在 [Astro Preact example](https://github.com/withastro/astro/tree/latest/examples/framework-preact) 中展示了如何在 Astro 项目中使用交互式的 Preact 组件。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:125]:
```suggestion
*   在 [Astro Preact 示例](https://github.com/withastro/astro/tree/latest/examples/framework-preact) 中展示了如何在 Astro 项目中使用交互式的 Preact 组件。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+*   在 [Astro Nanostores example](https://github.com/withastro/astro/tree/latest/examples/with-nanostores) 中展示了如何在 Astro 项目中在不同组件之间共享状态，甚至可以跨不同框架共享状态。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:126]:
```suggestion
*   在 [Astro Nanostores 示例](https://github.com/withastro/astro/tree/latest/examples/with-nanostores) 中展示了如何在 Astro 项目中在不同组件之间共享状态，甚至可以跨不同框架共享状态。
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+
+## 问题排查
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:128]:
```suggestion
## 故障排查
```

File: src/content/docs/zh-cn/guides/integrations-guide/preact.mdx [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+
+如果需要帮助，请访问 [Discord](https://astro.build/chat) 上的 `#support` 频道。我们友好的支持团队成员将在那里帮助你！
+
+你也可以查阅Astro集成文档 [Astro Integration Documentation][astro-integration] 获得更多集成信息。
+
+##  贡献 
+
+这个包由 Astro 的核心团队维护。欢迎你提交问题或（PR）！
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/preact.mdx:136]:
```suggestion
这个包由 Astro 的核心团队维护。欢迎你提交 isse 或 PR!
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
Thanks for the PR, here are some suggested changes to merge once the changes are complete.

**Comment by lu-sd** [comment]
Thanks for the review!

**Comment by Yan-Thomas** [comment]
Thanks @lu-sd! I can see there are a few suggestions in "Files changed" that still show up as not addressed, could you take a look at these in case you missed applying them?

**Comment by lu-sd** [comment]
oops.  Updated. Thanks!

**Review by liruifengv — APPROVED** [review]
LGTM! So cool! We've done it!!!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the hard work everyone! :rocket:

**Pull request merged**